.circleci/config.yml

```
@@ -229,17 +228,17 @@ jobs:
                   git push origin "$tag"
             - run:
                 name: Create GitHub release
-                # Implicit usage of ${GITHUB_TOKEN} by hub
+                # Implicit usage of ${GITHUB_TOKEN} by gh
                 # language=sh
                 command: |
                   source scripts/utils.sh
                   corresponding_milestone_title=v$(extract_version_from_current_wd)
                   tag=$corresponding_milestone_title
                   change_notes=$(./gradlew getChangelog --no-header -q)
-                  hub release create \
-                    --attach build/distributions/git-machete-intellij-plugin-*.zip \
-                    --message "$tag"$'\n\n'"$change_notes" \
-                    $tag
+                  gh release create \
+                    $tag \
+                    "$(./gradlew -q printSignedPluginZipPath)" \
```

```suggestion
                    "$(./gradlew -q $printSignedPluginZipPath)" \
```
isn't `printSignedPluginZipPath` a variable?